src/android/plugin/google/maps/GoogleMaps.java

```diff
@@ -192,9 +192,9 @@ public void run() {
             e.printStackTrace();
           }
        */
-        if (Build.VERSION.SDK_INT >= 21 || "org.xwalk.core.XWalkView".equals(view.getClass().getName())){
```

Comment:
I'm trying to figure out why they would do this...but I dunno...
Have we tested this with several devices? It's like they are checking that the webview is one of the standard class ( not Crosswalk!)